extra-modules/extra-mappings/src/main/kotlin/com/kotlindiscord/kord/extensions/modules/extra/mappings/arguments/MappingArguments.kt

```diff
@@ -18,4 +19,10 @@ open class MappingArguments(val namespace: Namespace) : Arguments() {
         true,
         namespace
     )
+
+    open val mapDescriptors by defaultingBoolean(
+        "mapDescriptors",
```

Comment:
This is user-facing, it shouldn't use programmer casing. `kebab-case` is standard.